conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceHandlerGenerator.java

```diff
@@ -405,6 +408,9 @@ private TypeName immutableCollection(TypeName input) {
             } else if (SET_NAME.equals(parameterized.rawType)) {
                 return ParameterizedTypeName.get(
                         IMMUTABLE_SET_NAME, parameterized.typeArguments.toArray(new TypeName[0]));
+            } else if (MAP_NAME.equals(parameterized.rawType)) {
+                return ParameterizedTypeName.get(
+                        IMMUTABLE_MAP_NAME, parameterized.typeArguments.toArray(new TypeName[0]));
```

Comment:
This appears to have been an oversight in https://github.com/palantir/conjure-java/pull/1051, so I've added it here.